include/ppx/bitmap.h

```diff
@@ -192,6 +195,11 @@ class Bitmap
     void   InternalCtor();
     Result InternalInitialize(uint32_t width, uint32_t height, Bitmap::Format format, uint32_t rowStride, char* pExternalStorage);
     Result InternalCopy(const Bitmap& obj);
+    void   FreeStbiDataIfNeeded();
+
+    // Stbi-specific functions/wrappers
+    static char*  StbiLoad(const std::filesystem::path& path, Bitmap::Format format, int* width, int* height, int* channels, int requiredChannels);
```

Comment:
I think the project convention is to name these pWidth, etc

Reply:
Done.